manifests/m/Microsoft/msodbcsql/18.3.3.1/Microsoft.msodbcsql.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+PackageIdentifier: Microsoft.msodbcsql
+PackageVersion: 18.3.3.1
+PackageLocale: en-US
+Publisher: Microsoft
+PublisherUrl: https://docs.microsoft.com/en-us/sql/connect/odbc/microsoft-odbc-driver-for-sql-server?view=sql-server-ver15
+PackageName: ODBC Driver 18 for SQL Server
+License: Copyright (c) Microsoft Corporation
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate